lib/ramble/ramble/application.py

```diff
@@ -778,6 +778,10 @@ def _inputs_and_fetchers(self, workload=None):
             for input_file in workload['inputs']:
                 input_conf = self.inputs[input_file]
 
+                # Expand input value as it may be a var
+                expanded_url = self.expander.expand_var(input_conf['url'])
```

Comment:
I'm not sure if this will actually matter, but would it be safer to create a copy of `input_conf` and overwrite that?

Just to make sure the change is constrained to this specific instance?

Reply:
Sure. Safety first! However, if that's the case then maybe it's better to do something more like:

```
                input_conf['expanded_url'] = expanded_url
                fetcher = ramble.fetch_strategy.URLFetchStrategy(**input_conf)
                file_name = os.path.basename(input_conf['expanded_url'])
```
(ie just had the file name consume the expanded one?)

Reply:
I think you'll need `input_conf['url']` to be set to the expanded one too, so that the fetcher is configured with the right URL.

Reply:
Sure, makes sense. I can add a `.copy()`